Gemfile

```diff
@@ -8,10 +8,15 @@ gem 'sinatra'
 gem 'sinatra-contrib'
 gem 'slim'
 gem 'rack'
-gem 'rack-protection'
+# update for security issue reported by github
+gem 'rack-protection', '~> 1.5.5'
```

Comment:
Do we need to put the 1.5.5 version here?

Reply:
That is the safest thing to do.  It minimizes the changes in dependencies.  If this was a big release with full testing then upping to the most recent version would be reasonable but it is better not do tempt fate.